packages/embed/README.md

```diff
@@ -255,6 +256,9 @@ You can listen to form events by providing callback methods:
 
       // for plans with "Redirect from ending screen" feature you also receive `ref`:
       console.log(`Ending button clicked in end screen ${ref}`)
+    },
+    onDuplicateDetected: ({ formId }) => {
```

Comment:
Thanks for adding an example here 👍 

Please add one to the [packages/demo-html/public/callbacks.html](https://github.com/Typeform/embed/blob/main/packages/demo-html/public/callbacks.html#L16-L39) as well, so we have a runnable example.

Reply:
@mathio Do you happen to have access to the test form? We'd need to enable the duplicate prevention feature there.

Reply:
I dont think I do :( Actually, it might be better to create a new form and new HTML page for demo purposes, since this feature would affect other demos when enabled.